packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -0,0 +1,1005 @@
+								</Text>
+							) }
+						{ commentDeleteMessage &&
+							thread.id === showConfirmationTabId && (
+								<Text>
+									{ __( 'Thread deleted successfully.' ) }
+								</Text>
+							) }
+						{ replyMessage &&
+							thread.id === showConfirmationTabId && (
+								<Text>
+									{ __( 'Reply added successfully.' ) }
+								</Text>
+							) }
+
+						{ showConfirmation &&
+							thread.id === showConfirmationTabId && (
+								<VStack
+									title={ __( 'Confirm' ) }
+									onRequestClose={ () =>
+										setShowConfirmation( false )
+									}
+									className="editor-collab-sidebar__useroverlay confirmation-overlay"
+									spacing="0"
+									justify="space-between"
+								>
+									<p>
+										{ __(
+											'Are you sure you want to mark this thread as resolved?'
+										) }
+									</p>
+									<HStack>
+										<Button
+											variant="primary"
+											onClick={ () =>
+												confirmAndMarkThreadAsResolved(
+													thread.id
+												)
+											}
+										>
+											{ __( 'Yes' ) }
+										</Button>
+										<Button
+											onClick={ () =>
+												setShowConfirmation( false )
+											}
+										>
+											{ __( 'No' ) }
+										</Button>
+									</HStack>
+								</VStack>
+							) }
+
+						{ deleteCommentShowConfirmation &&
+							thread.id === showConfirmationTabId && (
+								<VStack
+									title={ __( 'Confirm' ) }
+									onRequestClose={ () =>
+										setDeleteCommentShowConfirmation(
+											false
+										)
+									}
+									className="editor-collab-sidebar__useroverlay confirmation-overlay"
+									spacing="0"
+									justify="space-between"
+								>
+									<p>
+										{ __(
+											'Are you sure you want to delete this thread?'
+										) }
+									</p>
+									<HStack>
+										<Button
+											variant="primary"
+											onClick={ () =>
+												confirmDeleteComment(
+													thread.id
+												)
+											}
+										>
+											{ __( 'Yes' ) }
+										</Button>
+										<Button
+											onClick={ () =>
+												setDeleteCommentShowConfirmation(
+													false
+												)
+											}
+										>
+											{ __( 'No' ) }
+										</Button>
+									</HStack>
+								</VStack>
+							) }
+
+						{ 0 < thread?.reply?.length &&
+							thread.reply.map( ( reply ) => (
+								<VStack
+									key={ reply.id }
+									className="editor-collab-sidebar__childThread"
+									id={ reply.id }
+									spacing="2"
+								>
+									<HStack
+										alignment="left"
+										spacing="3"
+										justify="flex-start"
+									>
+										<img
+											src={
+												reply
+													?.author_avatar_urls?.[ 48 ]
+											}
+											className="editor-collab-sidebar__userIcon"
+											alt={ __( 'User avatar' ) }
+											width={ 32 }
+											height={ 32 }
```

Comment:
These sizes will need to be declared centrally, and perhaps made configurable / filterable.

Reply:
Absolutely, we can discuss that. However, since this is the initial phase, we are currently keeping the size hardcoded to maintain the desired look and feel.